src/ImageSharp/Common/Helpers/DenseMatrixUtils.cs

```diff
@@ -86,6 +87,7 @@ internal static class DenseMatrixUtils
             Span<ComplexVector4> targetRow,
             int row,
             int column,
+            int minRow,
```

Comment:
Maybe use `offsetRow` instead and reorder the arguments to gave a more clear signature?

Reply:
Isn't `minRow` more coherent though?
There's already a `maxRow` parameter that does exactly the same, just on the opposite side of the target rectangle, and that is used in the `Clamp` method.
This way it is called with `Clamp(minRow, maxRow)` which makes it extremely clear.
If I used `offsetRow` it'll be `Clamp(offsetRow, maxRow)`, which to me is much less intuitive, as each parameter has a different name there for no apparent reason.

Reply:
Using `minRow` and `offsetColumn` just looked a bit confusing. Could you not use `min` or `offset` for both?

Reply:
I see what you mean now, I hadn't noticed the name mismatch in the column paarameters since those were the same from the already existing convolution methods, I'll go ahead and make them all use `min`/`max` 👍

Also, can you mark this PR as ready for review? I don't see the option to change the PR state on my end, and waas wondering if you mods could do that for me.